src/engine/sparqlExpressions/DateExpressions.cpp

```diff
@@ -53,6 +59,23 @@ inline auto extractStrTimezone =
   return Id::makeUndefined();
 };
 
+//______________________________________________________________________________
+inline auto extractTimezoneDurationFormat =
```

Comment:
I think the correct implementation is not
to return an `LiteralOrIri`, but to return an `ID` which then stores the `dayTimeDuration`.
I think the exporting to string already does the right thing (because you implement the `toStringAndType` function.
The only thing that is then still left for a full support of this datatype (xs:dayTimeDuration), 
is to integrate it into the TurtleParser (TurtleParser.cpp, just search for the XSD_.... datatypes, then you find the function that does the complete switch.